configure

```diff
@@ -25685,33 +25685,39 @@ fi
 
 for module in $ac_shared_modules ; do
   moduledir=`echo "$module" | sed -e 's/\.la$//'`;
+  modulename=`echo "$module" | sed -e 's/\.o$//'`;
+  src=`echo "$module" | sed -e 's/\.o$//'`.c;
+  srcinc=`echo "$module" | sed -e 's/\.o$//'`.h;
 
   if test -f $srcdir/modules/$src -o -f $srcdir/contrib/$src; then
     continue
 
   elif test -d $srcdir/modules/$moduledir; then
     ac_shared_module_dirs="$ac_shared_module_dirs modules/$moduledir";
-    ac_shared_modules=`echo "$ac_shared_modules" | sed -e "s/$module//"`
+    ac_shared_modules=`echo "$ac_shared_modules" | sed -e "s/$modulename\.o//"`
```

Comment:
We then use the new `$modulename` variable, _and explicitly add the `.o` suffix_ to be removed, with `.` character properly escaped.